README.md

```diff
@@ -27,26 +27,36 @@ or use the provided [requirements.txt](requirements.txt) file provided in this r
 ```
 pip install -r requirements.txt
 ```
-# Release Versioning Annotations  
-synapseAnnotations release version tags structure **v0.0.0** fits within [semantic versioning](http://semver.org/) guidelines. 
-The tags release numbering scheme is defined as: 
+
+## Proposing changes
+
+1. Make changes on your feature branch.
+1. Request and complete a review from someone on the team.
+1. When review is completed, note it to be reviewed and merged at the weekly meeting.
+1. Finalize merge into the `master` branch.
+1. Update the version and make a versioned release (with assistance from @teslajoy)
+
+# Release Versioning Annotations
+Releases are made through Github tags and are available on the [Releases](https://github.com/Sage-Bionetworks/synapseAnnotations/releases) page.
+
+The release version structure **v0.0.0** follows [semantic versioning](http://semver.org/) guidelines. New releases are made using the following rules:
 
 Major version **v0** increments by:
 1. Changes in data structure (ex. yaml to json or json to mongodb)
-2. Changes in keys 
-3. Changes in values 
+2. Changes to existing keys
+3. Changes to existing values
 
 Minor version **.0.** increments by: 
 1. Adding keys
-2. Adding values 
+2. Adding values
 
 Patch version **.0** increments by: 
-1. Errors that don't break the API 
+1. Errors or corrections that don't break the API
 
-To optimize usability, the release tags will be placed on 2 required and 1 optional locations: 
-1. A Synapse project's entity annotation as a single value vs. list of versions and defined by the key **annotationVersion** uniformly through synapse projects. 
-2. [Annotation UI](https://github.com/Sage-Bionetworks/annotationUI) **title** 
-3. OPTIONAL: **Project's wiki** 
+To optimize usability, the release tags should be placed on two required and one optional locations: 
```

Comment:
Is there a way to encode this in a json file so when we audit we can audit against a specific version?